ROMFS/px4fmu_common/init.d/airframes/4011_dji_f450

```diff
@@ -10,24 +10,10 @@
 
 sh /etc/init.d/rc.mc_defaults
 
-set MIXER quad_x
-set PWM_OUT 1234
-
 if [ $AUTOCNF = yes ]
 then
-	param set MC_ROLL_P 7
-	param set MC_ROLLRATE_P 0.15
-	param set MC_ROLLRATE_I 0.05
-	param set MC_ROLLRATE_D 0.003
-	param set MC_PITCH_P 7
-	param set MC_PITCHRATE_P 0.15
-	param set MC_PITCHRATE_I 0.05
-	param set MC_PITCHRATE_D 0.003
-	param set MC_YAW_P 2.8
-	param set MC_YAWRATE_P 0.3
-	param set MC_YAWRATE_I 0.1
-	param set MC_YAWRATE_D 0
-
```

Comment:
For explanation of these deletions, all of these are all default parameter values.